config/index.js

```diff
@@ -379,23 +373,57 @@ var conf = convict({
       format: 'port',
       doc: 'Port for Redis server'
     },
-    sessionsKeyPrefix: {
-      default: 'fxa-auth-session',
-      env: 'SESSIONS_REDIS_KEY_PREFIX',
-      format: String,
-      doc: 'Key prefix for session tokens in Redis'
-    },
-    maxConnections: {
-      default: 200,
-      env: 'REDIS_POOL_MAX_CONNECTIONS',
-      format: 'int',
-      doc: 'Maximum connection count for Redis'
+    sessionTokens: {
+      enabled: {
+        default: true,
+        doc: 'Enable Redis for session tokens',
+        format: Boolean,
+        env: 'USE_REDIS'
+      },
+      prefix: {
+        default: 'fxa-auth-session',
+        env: 'SESSIONS_REDIS_KEY_PREFIX',
+        format: String,
+        doc: 'Key prefix for session tokens in Redis'
+      },
+      maxConnections: {
+        default: 200,
+        env: 'REDIS_POOL_MAX_CONNECTIONS',
+        format: 'int',
+        doc: 'Maximum connection count for the session token Redis pool'
+      },
+      minConnections: {
+        default: 2,
+        env: 'REDIS_POOL_MIN_CONNECTIONS',
+        format: 'int',
+        doc: 'Minimum connection count for the session token Redis pool'
+      }
     },
-    minConnections: {
-      default: 2,
-      env: 'REDIS_POOL_MIN_CONNECTIONS',
-      format: 'int',
-      doc: 'Minimum connection count for Redis'
+    email: {
+      enabled: {
+        default: true,
+        doc: 'Enable Redis for email config',
+        format: Boolean,
+        env: 'EMAIL_CONFIG_USE_REDIS'
+      },
+      prefix: {
+        default: 'email:',
+        env: 'EMAIL_CONFIG_REDIS_KEY_PREFIX',
+        format: String,
+        doc: 'Key prefix for the email config Redis pool'
+      },
+      maxConnections: {
+        default: 10,
+        env: 'EMAIL_CONFIG_REDIS_POOL_MAX_CONNECTIONS',
+        format: 'int',
+        doc: 'Maximum connection count for the email config Redis pool'
+      },
+      minConnections: {
+        default: 1,
```

Comment:
Redis access for the email config won't be happening anything like as frequently as it does for session tokens (mostly because of `/certificate/sign`), so these values can be much lower here. And there's only reads involved anyway, so we don't have to worry about concurrent updates which is the problem with the session token stuff.